ozaria/site/styles/play/variables.sass

```diff
@@ -0,0 +1,3 @@
+$api-bar-width: 279px
```

Comment:
This has been moved from legacy to ozaria folder, with the only change in the value of $api-bar-width